src/test/regress/expected/multi_large_table_task_assignment.out

```diff
@@ -186,17 +185,21 @@ SET citus.large_table_shard_count TO 2;
 DEBUG:  StartTransactionCommand
 DEBUG:  ProcessUtility
 DEBUG:  CommitTransactionCommand
--- The next test, dual hash repartition join, uses the current jobId to assign
--- tasks in a round-robin fashion. We therefore need to ensure that jobIds start
--- with an odd number here to get consistent test output.
-SELECT case when (currval('pg_dist_jobid_seq') % 2) = 0
-       	    then nextval('pg_dist_jobid_seq') % 2
-	    else 1 end;
+-- bump the job ID to an odd number
```

Comment:
This deletes a bit much, it's not clear without the leading test why we'd do anything about an odd number (nor that there's actually any bumping going on?)